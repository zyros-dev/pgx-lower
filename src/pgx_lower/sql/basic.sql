LOAD 'pgx_lower.so';
SELECT 'hello';
CREATE TABLE test(id SERIAL);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (1);
INSERT INTO test(id) VALUES (16);
SELECT * FROM test;
