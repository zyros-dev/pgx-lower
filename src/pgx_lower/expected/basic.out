LOAD 'pgx_lower.so';
NOTICE:  Installing custom executor hook...
SELECT 'hello';
NOTICE:  Custom executor is being executed in C! Ahah!
NOTICE:  Calling C++ executor from C...
NOTICE:  Inside C++ executor! Plan type: 1
NOTICE:  Query text: SELECT 'hello';
 ?column? 
----------
 hello
(1 row)

